### PR TITLE
Fix/improve language

### PR DESCRIPTION
**Pull request by salim-b**

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -90,7 +90,7 @@ <h3 class="mt-0">Encryption</h3>
                   All of your one-time passwords are stored in a vault. If you
                   choose to set a password (highly recommended), the vault will
                   be encrypted using strong cryptography. If someone with
-                  malicious intent gets a hold of the vault file, it’s
+                  malicious intents gets a hold of the vault file, it’s
```
Comment (kurtmckee) [inline layouts/index.html:93]: I think that "intent" is correct here.

File: layouts/index.html [hunk 2]
```diff
@@ -106,13 +106,14 @@ <h3 class="mt-0">Encryption</h3>
               <div class="feature-extended-body">
                 <h3 class="mt-0">Organization</h3>
                 <p>
-                  Over time, you'll likely accumulate tens of entries in your
+                  Over time, you'll likely accumulate dozens of entries in your
                   vault. Aegis Authenticator has lots of organization options to
                   make finding the one you need at a particular moment easier.
-                  Set a custom icon for an entry to make it easier to find.
+                  Set a custom icon for an entry to make it easier to find it.
```
Comment (kurtmckee) [inline layouts/index.html:112]: "it" does not need to be added here; the entry is already referenced by "make _it_ easier to find".

**Review by kurtmckee — CHANGES_REQUESTED** [review]
I think several of the changes here look good, but a few should be removed.

**Comment by alexbakker** [comment]
Thanks, but these changes are no longer relevant for the redesign of the website.

**Pull request closed without merging**